artemis-web/src/main/java/org/apache/activemq/artemis/component/WebServerComponent.java

```diff
@@ -258,6 +277,10 @@ private ServerConnector createServerConnector(HttpConfiguration httpConfiguratio
                }
             }
          }
+         if (binding.getSslAutoReload() != null && binding.getSslAutoReload() == true) {
```

Comment:
Perhaps replace the 2 checks with simply Boolean.TRUE.equals(binding.getSslAutoReload()) ?